scripts/configFromDB.py

```diff
@@ -251,6 +251,11 @@
         out.add_section(pname)
         for d in dir(p['process']):
             if d[0] == '_': continue
+            if d == 'output_product':
+                product_id = str(getattr(p['process'], d))
+                product_name  = dbu.getEntry('Product', product_id).product_name
+                out.set(pname, d, 'product_' + product_name)
+                continue
```

Comment:
This seems unrelated to the postgres changes. This looks to be adding "product_" to the output product field of every process created from the database. That'll make it match the section name that's created for the product, but in theory that's not supposed to matter: the output_product should be specifiable by either the section name or the actual product name, which this would be *without* the leading `product_`.

Did this crop up as a problem? If so, we should look into addFromConfig, the bug would be there (in not recognizing the product name instead of just the section name).

Reply:
That is correct,  every product starts with "product_" and every process starts with "process_"  This is to match the config file with the database.  

If you can crop "product_"  make addFromConfig.py works with dbprocessing that be better in my opinion.   The code was already written this way and to make it work with some sections we have here.   I remember working on findiing "process_"

Reply:
Okay, I'll double-check this; it *should* work with the actual product/process name, not just the section heading.

Reply:
This is the documentation I have for the config file
```
# Honored database substitutions used as {Y}{MILLI}{PRODUCT}
#	Y: 4 digit year
#	m: 2 digit month
#	b: 3 character month (Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)
#	d: 2 digit day
#	y: 2 digit year
#	j: 3 digit day of year
#	H: 2 digit hour (24-hour time)
#	M: 2 digit minute
#	S: 2 digit second
#	MILLI: 3 digit millisecond
#	MICRO: 3 digit microsecond
#	QACODE: the QA code (ok|ignore|problem)
#	VERSION: version string, interface.quality.revision
#	DATE: the UTC date from a file, same as Ymd
#	MISSION: the mission name from the db
#	SPACECRAFT: the spacecraft name from the db
#	PRODUCT: the product name from the db


##########################
# MANUAL
#
# Loops over the configuration file and if the DB does not have the mission,
# satellit, instrument entries present they are added, skipped otherwise.
# the Product and process entries must be unique and not present.
#
# THERE IS CURRENTLY NO UPDATE IN THE DB BASED ON THIS CONFIG SCRIPT


##################
# Required elements
#
# [mission]  <- once and only once with
#   rootdir  (string)
#   mission_name  (string)
#   incoming_dir  (string)
# [satellite] <- once and only once with
#   satellite_name  (string)
# [instrument] <- once and only once with
#   instrument_name  (string)
##### products and inspector are defined together since they are a one-to-one
# [product] <- multiple entries each starting with "product" then a unique identifer
#   product_name  (string)
#   relative_path  (string)
#   level  (float)
#   format  (string)
#   product_description (string)
#   inspector_filename (string)
#   inspector_relative_path (string)
#   inspector_description (string)
#   inspector_version (version e.g. 1.0.0)
#   inspector_output_interface (integer)
#   inspector_active (Boolean e.g. True or 1 or False or 0)
#   inspector_date_written (date e.g. 2013-07-12)
#   inspector_newest_version  (Boolean e.g. True or 1 or False or 0)
#   inspector_arguments (string)
#### processes and codes operate on the names of the products, they can be in
#### this config file or already in the db codes are one-to-one with processes
# [process] <- multiple entries each starting with "process" then a unique identifer
#   process_name (string)
#   output_product (string)  - identifer from section heading
#   output_timebase  (string, FILE/DAILY/WEEKLY/MONTHLY/YEARLY)
#   extra_params (string)
## A collection of input names entered as such
## the required portion is "optional_input" or "required_input" then some
## unique identifer on the end
#   optional_input1  (string) name of product - identifer from section heading
#   optional_input2  (string) name of product - identifer from section heading
#   optional_input3  (string) name of product - identifer from section heading
#   required_input1  (string) name of product - identifer from section heading
#   required_input2  (string) name of product - identifer from section heading
#   required_input3  (string) name of product - identifer from section heading
## code is entered as part of process
#   code_filename (string)
#   code_relative_path (string)
#   code_start_date (date, 2000-01-01)
#   code_stop_date  (date, 2050-12-31)
#   code_description (string)
#   code_version  (version e.g. 1.0.0)
#   code_output_interface  (integer)
#   code_active (Boolean e.g. True or 1 or False or 0)
#   code_date_written   (date, 2050-12-31)
#   code_newest_version (Boolean e.g. True or 1 or False or 0)
#   code_arguments (string)
```

Reply:
Ah, I see right now configFromDB does use the section name not the product name for the inputs, so doing the same thing for the outputs makes sense. I'll leave this and opened #81 as an enhancement for this.